RELEASE_NOTES.md

```diff
@@ -1,10 +1,13 @@
+### Version 1.2.2
+__Changes__
+- read 'functions' field in view functional profile apps (in KBaseGenomes.Genome-9.0 data typedef)
```

Comment:
Should also mention removal of `trim_tree_to_genomeSet()`

Reply:
added